### PR TITLE
More tooltips

### PR DESCRIPTION
**Pull request by brumm**
Hey,

I went ahead and added a couple of tooltips for some icon-only buttons that confused me the first time I used codesandbox.

I didn't investigate https://github.com/CompuIves/codesandbox-client/commit/2b9209fd6f00238dc4ebf2745fa51d017f08dbdf for too long, but it fixed the scrollbar issue for me.

### CODE REVIEW

**Comment by CompuIves** [comment]
Niiice! Will test it once I get home.

File: src/app/pages/Sandbox/Editor/Content/Header/index.js [hunk 1]
```diff
@@ -297,6 +314,7 @@ export default class Header extends React.PureComponent<Props> {
             ) : (
               <Action
                 onClick={userActions.signIn}
+                tooltip="Sign in with Github"
```
Comment (Tushkiz) [inline src/app/pages/Sandbox/Editor/Content/Header/index.js]: I think a tooltip here will be redundant as the button has text 🤔

File: src/app/components/sandbox/ModeIcons.js [hunk 2]
```diff
@@ -131,22 +131,28 @@ const getCurrentMode = ({
   setPreviewView,
 }: Props) => {
   const both = (
-    <ViewIcon onClick={setMixedView} active={showEditor && showPreview}>
-      <EditorIcon half />
-      <PreviewIcon half />
-    </ViewIcon>
+    <Tooltip title="Show both">
```
Comment (Tushkiz) [inline src/app/components/sandbox/ModeIcons.js]: Maybe you can make these ModeIcon tooltip titles consistent with those ones [below](https://github.com/CompuIves/codesandbox-client/pull/239/files#diff-1f317ba5bd25c442a32513566177d0c3R264). Currently we have them as `Editor view`, `Split view` and `Preview view`

File: src/app/pages/Sandbox/Editor/Content/Header/index.js (hunk 1 above)
Comment (brumm) [reply src/app/pages/Sandbox/Editor/Content/Header/index.js]: Good point, will remove.

File: src/app/components/sandbox/ModeIcons.js (hunk 2 above)
Comment (brumm) [reply src/app/components/sandbox/ModeIcons.js]: Ah, I didn't notice those.

I'd like to suggest moving from _this is the name of the thing you're clicking on_ to a more verb-oriented  _this is what will happen when you click_ style of descriptions. I'd like to change the existing Tooltips to reflect my changes, if possible.

Comment (Tushkiz) [reply src/app/components/sandbox/ModeIcons.js]: Let's keep the existing unchanged and change ours to `Show Editor view`, `Show Split view` and `Show Preview view`. What do you say?

Comment (brumm) [reply src/app/components/sandbox/ModeIcons.js]: I like it! Will do.

**Comment by CompuIves** [comment]
**Great**! Thanks @brumm!

**Pull request merged**